src/ark/utils/data_utils.py

```diff
@@ -220,21 +294,27 @@ def generate_and_save_cell_cluster_masks(fovs: List[str],
             Specify what to append at the end of every cell mask. Defaults to `''`.
     """
 
+    ccmd = CellClusterMaskData(
+        data=cell_data,
+        fov_col=settings.FOV_ID,
```

Comment:
Same comment as above, can we have `fov_col` be an argument to `generate_and_save_cell_cluster_masks` and make the default `settings.FOV_ID`?